Vagrantfile

```diff
@@ -7,49 +7,26 @@ VAGRANTFILE_API_VERSION = "2"
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   vms = [
     {
-      :id => "debian-buster-amd64",
-      :box => "debian/buster64",
+      :id => "debian-bullseye",
+      :box => "bento/debian-11.3",
     },
```

Comment:
bento/debian-11.3 is a bit old point release, how about using bento/debian-11?
at least we can test with debian 11.6 point release.

Reply:
Except above issue, LGTM.

Reply:
Thanks! I will check it!

Reply:
Fixed!
Thanks!